node/middlewares/viewerCountry.ts

```diff
@@ -0,0 +1,15 @@
+import { NotFoundError } from '@vtex/api'
+
+export async function viewerCountry(ctx: Context) {
+  const {
+    request: { header },
+  } = ctx
+
+  try {
+    const countryCode = header['cloudfront-viewer-country']
```

Comment:
render-server checks [two headers](https://github.com/vtex/render-server/blob/0f5e67febb91c0395c0dc841fdf079bb067113cd/node/middlewares/state.ts#L141). Should we do the same?

Reply:
I'll add it as well.